### ISSUE TITLE
Delete or update the kaitai_struct_visualizer_java.iml file

### ISSUE THREAD

**Issue by generalmimon** [kaitai-io/kaitai_struct_gui#22]
When I see the following:

https://github.com/kaitai-io/kaitai_struct_gui/blob/ae342c482cf9273da28640acf1c51a031ed02438/kaitai_struct_visualizer_java.iml#L15-L16

it is a clear sign that something needs to be done with this file. IIRC, nowadays it is automatically regenerated by IntelliJ IDEA anyway, so it would be best to remove it and Git should ignore all `*.iml` files.

**Comment by dgelessus** [comment]
Yep, the .idea directory and *.iml files should be deleted and put into .gitignore. We already have a Maven build, and as you say, IntelliJ uses that to automatically generate its own project files.

In my experience, having an existing IntelliJ project in the repo is actually *less* convenient, because then when you import the Maven build, the generated project files collide with the ones from the repo. At best you only get some unnecessary Git changes, at worst the resulting project behaves unexpectedly/wrong.
